modules/apiserversource-yaml.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="apiserversource-yaml_context"]
+= Using the ApiServerSource with the YAML method
+
+This guide describes the steps required to create, manage and delete an ApiServerSource using YAML files.
+
+.Prerequisites
+
+* You will need to have a Knative Serving and Eventing installation.
+* You will need to have created the `default` broker in the same namespace as the one defined in the ApiServerSource YAML file.
+
+.Procedure
+
+. Create a service account, role and role binding for the ApiServerSource.
++
+You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
++
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default <1>
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: event-watcher
+  namespace: default <1>
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - get
+      - list
+      - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: k8s-ra-event-watcher
+  namespace: default <1>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: event-watcher
+subjects:
+  - kind: ServiceAccount
+    name: events-sa
+    namespace: default <1>
+----
++
+<1> Adjust this namespace to the namespace that you have selected for installing ApiServerSource.
++
+[NOTE]
+====
+If you want to re-use an existing service account with the appropriate permissions, you must modify the `authentication.yaml` for that service account.
+====
++
+After you have created the `authentication.yaml` file, apply it by entering the following command:
++
+----
+$ oc apply --filename authentication.yaml
+----
+
+. Create an ApiServerSource event source.
++
+You can do this by creating a file named `k8s-events.yaml` and copying the following sample code into it:
++
+----
+apiVersion: sources.knative.dev/v1alpha1
+kind: ApiServerSource
+metadata:
+  name: testevents
+spec:
+  serviceAccountName: events-sa
+  mode: Resource
+  resources:
+    - apiVersion: v1
+      kind: Event
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1beta1
+      kind: Broker
+      name: default
+----
++
+After you have created the `k8s-events.yaml` file, apply it by entering the following command:
++
+----
+$ oc apply --filename k8s-events.yaml
+----
+
+. To check that the ApiServerSource is set up correctly, create a Knative service that dumps incoming messages to its log.
++
+Copy the following sample YAML into a file named `service.yaml`:
++
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display
```

Comment:
Should use a specific tag for the current release

quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2